R/collect_n_subject.R

```diff
@@ -38,14 +39,16 @@
 #' n_subject(r2rtf_adae$USUBJID, r2rtf_adae$TRTA)
 #' n_subject(r2rtf_adae$USUBJID, r2rtf_adae$TRTA, r2rtf_adae$SEX)
 #' n_subject(r2rtf_adae$USUBJID, r2rtf_adae$TRTA, r2rtf_adae$SEX, use_na = "always")
+#' n_subject(r2rtf_adae$USUBJID, r2rtf_adae$TRTA, r2rtf_adae$SEX, na = "Null")
```

Comment:
It looks like `na =...` is not a defined argument...?

Reply:
I just change it from the function argument.